src/tests/api/BavHappyPath.test.ts

```diff
@@ -16,22 +17,53 @@ describe("Test BAV End Points", () => {
 
     beforeEach(async () => {
         //Session Request
-        sessionId = await startStubServiceAndReturnSessionId(bavStubPayload);
+       sessionId = await startStubServiceAndReturnSessionId(bavStubPayload);
     });
     it("E2E BAV End Points Happy Path Journey", async () => {
         expect(sessionId).toBeTruthy();
 
-        // Make sure authSession state is as expected - BAV_SESSION_CREATED
+        // Make sure authSession state is as expected
         await getSessionAndVerifyKey(sessionId, constants.DEV_BAV_SESSION_TABLE_NAME, "authSessionState", "BAV_SESSION_CREATED");
 
         // Make sure txma event is present & valid
         const sqsMessage = await getSqsEventList("txma/", sessionId, 1);
         await validateTxMAEventData(sqsMessage);
+        
+        // // Commented out until /token (KIWI-1260) and /userInfo (KIWI-1258) endpoints are available
+        // // Token
+        // const tokenResponse = await tokenPost(authResponse.data.authorizationCode.value, authResponse.data.redirect_uri);
+        // expect(tokenResponse).toBe(200);
+        // // User Info
+        // const userInfoResponse = await userInfoPost("Bearer " + tokenResponse.data.access_token);
+        // expect(userInfoResponse).toBe(202);
+    });
+});
 
-        // Commented out until /authorisation (KIWI-1259), /token (KIWI-1260) and /userInfo (KIWI-1258) endpoints are available
-        // // Authorization
-        // const authResponse = await authorizationGet(sessionId);
-        // expect(authResponse.status).toBe(200);
+describe("Test BAV End Points", () => {
```

Comment:
Can we have all these tests under 1 description please, sharing the same beforeEach step as this is a pre-requisite for all happy path tests and differentiate in the it statement please? 

e.g. 

- Both tests can be under the same description

`describe("BAV CRI Happy Path", () => {`

- The beforeEach logic can stay the same 
- Differentiate between tests in the it step 
/session: `    it("BAV CRI /session Happy Path", async () => {`
/authorization: `    it("BAV CRI /authorization Happy Path", async () => {`